### ISSUE TITLE
NTR: Pyrroline-3-hydroxy-5-carboxylate high in urine (Lab: urine tests)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8844]
New term request Pyrroline-3-hydroxy-5-carboxylate high in urine (elthat_111112205604):   UMLS: 1855915

**Comment by pnrobinson** [comment]
@MickeySegal
is this https://omim.org/entry/239510?search=Pyrroline%20carboxylate&highlight=carboxylate%2Cpyrroline
Is the Delta missing?

**Comment by MickeySegal** [comment]
That is the disease, but our finding name is now " δ-1-pyrroline-3-hydroxy-5-carboxylate high in urine".  It seems there are many synonyms: https://hmdb.ca/metabolites/HMDB0001369

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000620
- label: Elevated urinary pyrroline hydroxycarboxylic acid level
- parents: [Elevated urinary carboxylic acid (HP:0040156)](https://hpo.jax.org/app/browse/term/HP:0040156)
- pmid: PMID:9700195
- def: The amount of pyrroline hydroxycarboxylic acid in the urine, normalized for urine concentration, is above the upper limit of normal.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8844
- synonyms: Elevated urinary 3-hydroxy-L-1-pyrroline-5-carboxylate level(NONE)|Elevated urine L-delta1-pyrroline 3-hydroxy-5-carboxylate level(NONE)
